content-test/index.js

```diff
@@ -9,6 +9,7 @@ describe("ActivtyStreams", () => {
   let restore;
   let originalAlert;
   let originalConfirm;
+  let originalServices;
```

Comment:
Leave it as `undefined`? Couldn't find the initialization/assignment of `originalService`.